docs/14_0_0/menu.md

```diff
@@ -8,6 +8,7 @@
   - [Hello World](/gettingstarted/helloworld.md)
   - [Whats New](/gettingstarted/whatsnew.md)
   - [Migration Guide](../migrationguide/migrationguide.md)
+  - [Community Projects](/gettingstarted/community.md)
```

Comment:
Can you move this up 1 line too I think the Migration Guide should be last so Community Projects can go under Whats New

Reply:
done